manifests/mod/auth_openidc.pp

```diff
@@ -4,7 +4,7 @@
 # @see https://github.com/zmartzone/mod_auth_openidc for additional documentation.
 #
 class apache::mod::auth_openidc (
-) inherits ::apache::params {
+) inherits apache::params {
```

Comment:
Should this be on line 6 given that we are not passing in anything to the constructor?

Reply:
Think it reads better this way though, looks neater in my eyes at least.